apps/wallet/src/ui/app/pages/accounts/AddAccountPage.tsx

```diff
@@ -82,9 +88,19 @@ export function AddAccountPage() {
 						size="tall"
 						text="Set up Ledger"
 						before={<LedgerLogo className="text-gray-90" width={16} height={16} />}
-						onClick={() => {
-							setConnectLedgerModalOpen(true);
+						onClick={async () => {
 							ampli.openedConnectLedgerFlow({ sourceFlow });
+							if (isPopup) {
+								const { origin, pathname, hash } = window.location;
+								await Browser.tabs.create({
+									url: `${origin}${pathname}${hash}${
+										hash.includes('showLedger') ? '' : `${hash.includes('?') ? '&' : '?'}showLedger`
```

Comment:
super nit: I would slightly prefer for us to make a new `URL` instance with the query param appended versus doing the manual processing of the query params even though it's a little more verbose

```
const urlWithLedgerModalOpen = new URL(`${origin}${pathname}${hash}`);
if (urlWithLedgerModalOpen.hash === '#showLedger') {
  urlWithLedgerModalOpen.searchParams.append('showLedger', '');
}
```

Reply:
I improved this [here](https://github.com/MystenLabs/sui/pull/13636/files#diff-2c52e928397f0dab0d0761a45a24c9f3a08c7e3dbd0770f70e19f0d05b4cf95fR23) in a following PR. I will skip updating this one to avoid conflicts